qemu/tests/win_virtio_driver_update_test.py

```diff
@@ -36,14 +47,14 @@ def run(test, params, env):
         error_context.context("Install virtio driver", logging.info)
     else:
         error_context.context("Downgrade virtio driver", logging.info)
-        new_params = params.copy()
-        new_params["cdrom_virtio"] = cdrom_virtio_downgrade
-        vm.create(params=new_params)
-        vm.verify_alive()
-        single_driver_install.run(test, new_params, env)
+        change_virtio_media(params["cdrom_virtio_downgrade"])
+        single_driver_install.run(test, params, env)
+        error_context.context("Reboot guest after downgrade virtio driver",
+                              logging.info)
+        vm.reboot(method="system_reset")
         error_context.context("Upgrade virtio driver to original",
```

Comment:
I think "shell reboot" is better than "system reset".

Reply:
agree with @kimi1978wy